packages/rancher-monitoring/overlay/app-README.md

```diff
@@ -0,0 +1,28 @@
+# Rancher Monitoring
+
+Modifies the upstream [Prometheus Operator](https://github.com/helm/charts/tree/master/stable/prometheus-operator) chart, which provides easy monitoring definitions for Kubernetes services and the deployment and management of Prometheus instances, and enables [Prometheus Adapter](https://github.com/helm/charts/tree/master/stable/prometheus-adapter) on a default Prometheus instance.
+
+You must install the Prometheus Operator CRDs first using the `rancher-monitoring-crd` chart before installing this chart. Once installed, this chart will contain the following components:
```

Comment:
nit: is there a link to this chart we can give here?

Reply:
Discussed this offline with Rajashree; we'll provide the `helm install` command to install the crd chart first instead since there is no physical repository to point to for the CRD chart.